cmd/relayd/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"net"
+
+	"github.com/elevran/relay/log"
+)
+
+const (
+	defaultAddress = ":8080"
+)
+
+func main() {
+
```

Comment:
why empty lines after the beginning of a function are used? I understand using empty lines between statements for logical grouping. An empty line before the first statement does not seem to convey a meaning of grouping